lib/kb_maxbin/kb_maxbinImpl.py

```diff
@@ -22,9 +22,9 @@ class kb_maxbin:
     # state. A method could easily clobber the state set by another while
     # the latter method is running.
     ######################################### noqa
-    VERSION = "1.0.0"
-    GIT_URL = "https://github.com/kbaseapps/kb_maxbin.git"
-    GIT_COMMIT_HASH = ""
+    VERSION = "1.0.2"
+    GIT_URL = "https://github.com/dcchivian/kb_maxbin"
```

Comment:
Since we're going to merge it back in, we should revert it back to using the kbaseapps URL

Reply:
sorry.  the make inserts that automatically

Reply:
will it get fix automagically or do you need to update and push?

Reply:
I just manually fixed it back now and pushed.  Should be part of PR now.